packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -653,6 +656,23 @@ class DropdownButton<T> extends StatefulWidget {
   /// [ThemeData.textTheme] of the current [Theme].
   final TextStyle style;
 
+  /// The widget to use for the drop-down button's icon.
+  ///
+  /// Defaults to an [Icon] with the [Icons.arrow_drop_down] glyph.
+  final Widget icon;
+
+  /// Sets the disabled icon color if an [Icon] widget is passed into the icon parameter.
```

Comment:
Developers might not understand "passed into the icon parameter".   Maybe: 

The color of any [Icon] descendant of [icon] if this button is disabled, i.e. if [onChanged] is null.